command.go

```diff
@@ -227,7 +227,7 @@ func (cmd *CommandFunc) Call(ctx context.Context, args, env []string) (int, erro
 		params = append(params, reflect.ValueOf(ctx))
 		x++
 	} else if ctx != context.TODO() {
-		panic("mixing usage of context aware and unaware commands")
+		panic("to use context, all commands must accept a context as their first argument")
```

Comment:
Should we use the word `context.Context` so there's ambiguity about which "context" we're talking about? (the term is fairly overloaded)

Reply:
Totally.  Will update.